### PR TITLE
Redo home page modules to reduce jargon

### PR DESCRIPTION
**Pull request by derekjuber**
Just my attempt so I'm open to criticisms.

The current home page modules (Build more, configure less & Manage complexity like a boss) are not as clear as they could be. I find the wording needlessly complex and the associated code for the second module is hard to follow why it helps me manage complexity since it's a huge createPlugin nested function. As a new user to Fusion, they will have no idea what's going on there.

I took this opportunity to redo the focus and text to make it less jargon'y and easier to read for a wider audience, one which may not be as senior to web frameworks. I chose to highlight things that make Fusion stand out, namely the plugin system and universal code/rendering, just in a simpler less complex way.

image

image

Also, Lucy pointed out that the bullet point lists at the bottom were too long so I culled the ones that were less important and less "sexy" for lack of a better word. So it's easier to digest.

This whole page could probably use a redesign but I'll leave that for a future task if we want to do that.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -251,31 +247,28 @@ export default () => {
...
-  provides: ({User, Project}) => ({
-    async findByProjectId(projectId) {
-      const {users} = await Project.findById(projectId);
-      return Promise.all(users.map(id => User.findById(id)));
-    },
-  }),
-});
+  // Register this plugin only on the server
+  if (__NODE__) {
+    app.register(SecureHeadersPlugin);
```
Comment (KevinGrandon) [inline src/pages/index.js]: I think that maybe `SecureHeadersPlugin` might confuse newcomers. I wonder  if we just have a simple inline plugin  that logs the current route or something.

Comment (derekjuber) [reply src/pages/index.js]: I just picked something that sounded server side’y and that wouldn’t be applicable on the client. Inlining code would add to the example code length also which I’m trying to avoid since the point of this section is about code fencing and not plugin code.

That being said I could maybe change it to something less jargony but I think SecureHeaders hopefully isn’t too complex of a term.
